src/patternfly/components/Dropdown/examples/Dropdown.md

```diff
@@ -173,11 +182,36 @@ The dropdown panel is provided for flexibility in allowing various content withi
 {{/dropdown}}
 ```
 
+```hbs title=Dropdown-with-avatar-and-info
+{{#> dropdown id="dropdown-with-avatar-and-info-example" dropdown--IsAvatarInfo="true" dropdown--IsExpanded="true"}}
+  {{> avatar avatar--attribute='src="/assets/images/img_avatar.svg" alt="Avatar image"'}}
+  {{#> dropdown-toggle-text}}
+    Ned Username
+  {{/dropdown-toggle-text}}
+  {{> dropdown-toggle-icon}}
+{{/dropdown}}
+```
+
+```hbs title=Dropdown-with-avatar-and-info-dark-theme
+<div class="pf-t-dark">
+{{#> dropdown id="dropdown-with-avatar-and-info-dark-theme-example" dropdown--IsAvatarInfo="true" dropdown--IsExpanded="true"}}
+  {{> avatar avatar--attribute='src="/assets/images/img_avatar.svg" alt="Avatar image"'}}
```

Comment:
I do think the image looks a little unbalanced here next to the text, even though if there was a photo it would be balanced. i think for a lot of users they would probably have this generic avatar, i wonder if design could add a border or a light background behind the person

Reply:
I do see what you mean @christiemolloy . But the reality is that this will only be used in the masthead where it will be against the dark background so I don't think that will be an issue.